lib/resty/healthcheck.lua

```diff
@@ -24,6 +24,7 @@
 -- @author Hisham Muhammad, Thijs Schreijer
 -- @license Apache 2.0
 
+local ngx = ngx
```

Comment:
The message for this commit has a typo (`optimzie`).

Reply:
fixed